lib/ini.js

```diff
@@ -12,6 +12,9 @@ const encode = (obj, opt = {}) => {
   /* istanbul ignore next */
   opt.platform = opt.platform || (typeof process !== 'undefined' && process.platform)
   opt.bracketedArray = opt.bracketedArray !== false
+  const commentsKey = opt.commentsKey || 'comments'
+  const commentsKeyRegEx = new RegExp(commentsKey, 'i')
```

Comment:
Is this really needed?  Building a regex from user input is not the best security practice.  Is it good enough to just say "it's comments" and let folks use that.  Barring that is there a way to define it as a string to match on without regex?